include/openssl/crypto.h

```diff
@@ -332,6 +332,9 @@ int FIPS_mode(void);
 int FIPS_mode_set(int r);
 
 void OPENSSL_init(void);
+void OPENSSL_fork_prepare(void);
+void OPENSSL_fork_parent(void);
+void OPENSSL_fork_child(void);
```

Comment:
(the other choice is not to guard them in `crypto/init.c`, but rather their bodies)